examples/platform/silabs/SiWx917/device/src/startup_common_RS1xxxx.c

```diff
@@ -0,0 +1,554 @@
+void WEAK IRQ075_Handler(void);                                           /*!<  WLAN Band1 intr0                 */
+void WEAK IRQ076_Handler(void);                                           /*!<  WLAN Band1 intr1                 */
+void WEAK IRQ077_Handler(void);                                           /*!<  Reserved                         */
+void WEAK IRQ078_Handler(void);                                           /*!<  Reserved                         */
+void WEAK IRQ079_Handler(void);                                           /*!<  BT intr                          */
+void WEAK IRQ080_Handler(void);                                           /*!<  ZB intr                          */
+void WEAK IRQ081_Handler(void);                                           /*!<  Reserved                         */
+void WEAK IRQ082_Handler(void);                                           /*!<  Modem disabled mode trigger intr */
+void WEAK IRQ083_Handler(void);                                           /*!<  gpio intr                        */
+void WEAK IRQ084_Handler(void);                                           /*!<  uart intr                        */
+void WEAK IRQ085_Handler(void);                                           /*!<  watch dog level intr             */
+void WEAK IRQ086_Handler(void);                                           /*!<  ULP Sleep sensor interrupt       */
+void WEAK IRQ087_Handler(void);                                           /*!<  ECDH intr                        */
+void WEAK IRQ088_Handler(void);                                           /*!<  DH intr                          */
+void WEAK IRQ089_Handler(void);                                           /*!<  QSPI intr                        */
+void WEAK IRQ090_Handler(void);                                           /*!<  ULP processor interrupt TASS     */
+void WEAK IRQ091_Handler(void);                                           /*!<  Sys Tick Timer                   */
+void WEAK IRQ092_Handler(void);                                           /*!<  Real Timer interrupt             */
+void WEAK IRQ093_Handler(void);                                           /*!<  PLL lock interrupt               */
+void WEAK IRQ094_Handler(void);                                           /*!<  Reserved                         */
+void WEAK IRQ095_Handler(void);                                           /*!<  UART2 Interrupt                  */
+void WEAK IRQ096_Handler(void);                                           /*!<  I2S Interrupt                    */
+void WEAK IRQ097_Handler(void);                                           /*!<  I2C Interrupt                    */
+void WEAK IRQ098_Handler(void);                                           /*!<  RESERVED                         */
+
+/*----------Symbols defined in linker script----------------------------------*/
```

Comment:
Yeah this files should be removed completely

Reply:
We cannot do modification in Wifi SDK startup file, we are seeing issues with that.